FreeRTOS-Plus/Source/FreeRTOS-Plus-TCP/portable/NetworkInterface/Zynq/NetworkInterface.c

```diff
@@ -1,5 +1,5 @@
 /*
- * FreeRTOS V202002.00
+ * FreeRTOS V202007.00

```

Comment:
I think we should make all the changes at once. I believe that this is not the only instance where the version number is old.
Should we drop this change?

Reply:
Shouldn't this have the same version number as the stack? https://github.com/FreeRTOS/FreeRTOS/blob/master/FreeRTOS-Plus/Source/FreeRTOS-Plus-TCP/FreeRTOS_TCP_IP.c#L2

Reply:
I just "undid" the version change:
~~~
- * FreeRTOS V202002.00
+ * FreeRTOS V202007.00
~~~
It is probably better to update all version numbers in a single PR.